ReleaseTooling/Sources/PodspecsTester/InitializeSource.swift

```diff
@@ -197,7 +197,7 @@ enum InitializeSpecTesting {
     return versionMatches[0][1]
   }
 
-  private static func createPodDirctory(specRepoPath: String, podName: String,
+  private static func createPodDirectory(specRepoPath: String, podName: String,
                                         version: String) -> URL {
```

Comment:
Looks like the indentation needs to be adjusted here to fix clang-format failure.

Reply:
thanks! 
I've adjusted the indentation to resolve the clang-format error. Let me know if anything else needs to be changed.

https://github.com/firebase/firebase-ios-sdk/pull/12793/commits/c2e03950d94b8dd8b62220bba36d874a74b8d4d4